stdlib/3/unittest/mock.pyi

```diff
@@ -1,4 +1,5 @@
 # Stubs for unittest.mock
+# Please sync with third_party/2and3/mock.pyi when changing.
```

Comment:
You can put this in `typeshed/tests/check_consistent.py` (you'll have to make sure the files are byte-for-byte identical).